plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/android/AndroidPluginsApplier.kt

```diff
@@ -69,7 +56,7 @@ private fun androidPluginConfigureAction(
 
         androidCommonExtension.sourceSets.all {
             // https://issuetracker.google.com/u/1/issues/170650362
-            val androidSourceSet = java as DefaultAndroidSourceDirectorySet
+            val androidSourceSet = sourceSet.kotlin as DefaultAndroidSourceDirectorySet
```

Comment:
this is a compile error currently. If you can fix this, I'd be happy to merge this PR.

Reply:
Fixed and rebased here https://github.com/JLLeitschuh/ktlint-gradle/pull/703/commits/eb47391571384f368901463c6d274e6bb3eb7b7b